tpm_quote/src/command.rs

```diff
@@ -0,0 +1,97 @@
+use crate::error::Error;
+
+use tss_esapi::{
+    handles::KeyHandle,
+    interface_types::{
+        algorithm::HashingAlgorithm, session_handles::AuthSession, structure_tags::AttestationType,
+    },
+    structures::{
+        Attest, AttestInfo, Data, HashScheme, PcrSelectionList, Signature, SignatureScheme,
+    },
+    Context,
+};
+
+use std::convert::TryFrom;
+
+pub(crate) fn attest_pcr(
```

Comment:
Is it possible to add a comment describing all checks made during the attestation?

Reply:
Added comments on public API `get_quote()` and `verify_quote()`.